site_root/input/resources/examples/Specimen-bs-001.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Can you fill out the `bodySite` in the example, just to make a little bit more complete.

Reply:
Updated and validated!